pkg/types/installconfig.go

```diff
@@ -294,3 +300,20 @@ type ImageContentSource struct {
 	// +optional
 	Mirrors []string `json:"mirrors,omitempty"`
 }
+
+// CloudCredentialsMode is the mode in which the cloud-credential-operator operates.
+// +kubebuilder:validation:Enum="";mint;passthrough;manual
+type CloudCredentialsMode string
```

Comment:
is `CloudCredentialsMode` really the accurate name given that installer installs to non cloud environment like vpshere openstack ovirt etc.

`CredentialsMode` woud be good enough?

Reply:
I thought **Cloud**CredentialsMode made more sense since it was driving the behavior of the **cloud**-credential-operator.

Reply:
the naming of operator might make sense that it can create and manage resources for cloud like platforms but the installer is not restricted to those platforms and this is a global field, so I would recommend we use a name that cover a more broader user-case

Reply:
OK. I will change it. My concern was that "Credentials" by itself may not be sufficient to determine what the credentials are for.